sentinel-extension/sentinel-datasource-spring-cloud-config/src/main/java/com/alibaba/csp/sentinel/datasource/spring/cloud/config/SentinelRuleLocator.java

```diff
@@ -134,7 +134,7 @@ public org.springframework.core.env.PropertySource<?> locate(
                 "Could not locate PropertySource and the fail fast property is set, failing",
                 error);
         }
-        RecordLog.warn("Could not locate PropertySource: " + (errorBody == null
+        RecordLog.warn("Could not locate PropertySource: {}" , (errorBody == null
```

Comment:
please remove the unnecessary indent before `,`

Reply:
There is no indentation here